Base/res/ladybird/about-pages/processes.html

```diff
@@ -83,14 +83,41 @@
                 maximumFractionDigits: 2,
             });
 
-            const memoryFormatter = new Intl.NumberFormat([], {
-                style: "unit",
-                unit: "byte",
-                notation: "compact",
-                unitDisplay: "narrow",
-                minimumFractionDigits: 2,
-                maximumFractionDigits: 2,
-            });
+            const memoryFormatters = [];
```

Comment:
Ohh I ran into this recently in #6675:
https://github.com/LadybirdBrowser/ladybird/pull/6675/commits/6130d33cf8fb0cadfb608e9ca985a43536ca57da#diff-e32009b90cf09eca69c20325482fdd357ef21351fc917e41b60cf722f6e31af5

I also found this `Math.log` based solution but ended up going with a simpler `/ 1024` loop, since it would be hit a max of like 5 times. I don't have a strong preference for which we use, but maybe we want to put this in a separate .js helper file, so that whatever we do, we do it consistently. (My usage of this is is still in a PR, so I'm happy to do that myself there).

Reply:
I've moved the memory formatting code into a separate helper file.